Source/Csla.Shared/Core/FieldManager/FieldDataManager.cs

```diff
@@ -1,7 +1,7 @@
 ﻿//-----------------------------------------------------------------------
 // <copyright file="FieldDataManager.cs" company="Marimer LLC">
 //     Copyright (c) Marimer LLC. All rights reserved.
-//     Website: https://cslanet.com
+//     Website: http://cslanet.com
```

Comment:
The website link was reverted to http. This was probably a mistake.

Reply:
Thanks, will be fixed in another PR